CLAUDE.md

```
@@ -0,0 +1,157 @@
+
+# Run linter
+npm run lint
+```
+
+## Architecture
+
+### Project Structure
+
+The project follows Next.js 15 App Router conventions with a custom organization pattern:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)